torch/_inductor/runtime/triton_heuristics.py

```diff
@@ -664,6 +664,10 @@ def kernel_call():
                 stream=stream,
             )
 
+        if do_bench_using_profiling:
+            from torch._inductor.utils import do_bench_using_profiling
+            return do_bench_using_profiling(kernel_call, warmup=10, rep=40)
+
```

Comment:
nit we probably shouldn't use the same variable name for both the bool and the benchmarking function, maybe `with_profiler=False` as the argument name?